AsyncDisplayKit/Details/ASCollectionDataController.mm

```diff
@@ -143,15 +145,14 @@ - (void)prepareForInsertRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
 
 - (void)willInsertRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
 {
-  NSArray *keys = _pendingContexts.allKeys;
-  for (NSString *kind in keys) {
+  for (NSString *kind in _pendingContexts) {
     NSMutableArray<ASIndexedNodeContext *> *contexts = _pendingContexts[kind];
```

Comment:
It's true. Scott says it's hard to know which API would be faster and that he wants to profile them head-to-head, so for now I decided to make the smallest possible change.